src/libraries/System.Private.Xml/src/System/Xml/Core/XmlParserContext.cs

```diff
@@ -34,17 +35,17 @@ public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string xmlLa
             // Intentionally Empty
         }
 
-        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string docTypeName,
-                  string pubId, string sysId, string internalSubset, string baseURI,
+        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string? docTypeName,
+                  string? pubId, string? sysId, string? internalSubset, string baseURI,
                   string xmlLang, XmlSpace xmlSpace)
         : this(nt, nsMgr, docTypeName, pubId, sysId, internalSubset, baseURI, xmlLang, xmlSpace, null)
         {
             // Intentionally Empty
         }
 
-        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string docTypeName,
-                          string pubId, string sysId, string internalSubset, string baseURI,
-                          string xmlLang, XmlSpace xmlSpace, Encoding enc)
+        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager? nsMgr, string? docTypeName,
+                          string? pubId, string? sysId, string? internalSubset, string baseURI,
+                          string xmlLang, XmlSpace xmlSpace, Encoding? enc)
```

Comment:
Actually all parameters except `XmlSpace xmlSpace` can be nullable for all overloads